xds/src/main/java/com/linecorp/armeria/xds/XdsBootstrap.java

```diff
@@ -69,6 +70,13 @@ static XdsBootstrap of(Bootstrap bootstrap, EventExecutor eventLoop) {
      */
     ListenerRoot listenerRoot(String resourceName);
 
+    /**
+     * Returns a new {@link ListenerRoot} that allows users to receive notifications when the resources
+     * in the tree are updated. The {@link HttpConnectionManager} in the {@link Listener} must have either,
+     * {@link HttpConnectionManager#hasRds()} or {@link HttpConnectionManager#hasRouteConfig()}.
```

Comment:
```suggestion
     * in the tree are updated.
```

Technically, it is perfectly valid to have a `Listener` without a `Route` attached to it

Reply:
> Technically, it is perfectly valid to have a Listener without a Route attached to it

Yes, but we cannot use the listener without a Route attached to it and 
it raises an exception if it doesn't have it. 
https://github.com/line/armeria/blob/e494846b2866df4a264442c6217b6d9ce76a36e4/xds/src/main/java/com/linecorp/armeria/xds/ListenerXdsResource.java#L56